flink-container/src/test/java/org/apache/flink/container/entrypoint/StandaloneJobClusterConfigurationParserFactoryTest.java

```diff
@@ -97,4 +103,27 @@ public void testSavepointRestoreSettingsParsing() throws FlinkParseException {
 		assertThat(savepointRestoreSettings.getRestorePath(), is(equalTo(restorePath)));
 		assertThat(savepointRestoreSettings.allowNonRestoredState(), is(true));
 	}
+
+	@Test
+	public void testSetJobIdManually() throws FlinkParseException {
+		final JobID jobId = new JobID();
+		final String[] args = {"--configDir", "/foo/bar", "--job-classname", "foobar", "--job-id", jobId.toString()};
```

Comment:
please also test the short option version `-jid`

Reply:
What are we testing with this? That the name `jid` does not change in the future? I don't want to test the library that we are using.

Reply:
if `-jid` is an accepted command-line argument then I would consider it as part of the API, in wbhicih case it should be covered by a test.

Is this just an artifact from the library we're using?